public/js/stores/index.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+export { default as app } from './app';
```

Comment:
I wonder if `export * from './app'` would be better as to delegate the named export to the module